.eslintrc.js

```diff
@@ -32,6 +33,7 @@ module.exports = {
         '@typescript-eslint/array-type': ['error', {
             default: 'array-simple',
         }],
+        'yoda': ['error', 'never'],
```

Comment:
Wait what, Yoda?
![image](https://user-images.githubusercontent.com/9638588/101485938-f7b4bb00-396c-11eb-9965-aef6588fee73.png)

Reply:
Yeah it's apparently for this line https://github.com/darkreader/darkreader/pull/4423/files#diff-aab4ceb6d3254a7b948f95d8057cd8d91cd43c09f95569885cbce6d8b079d926R154